mirage/helpers/get-all.js

```diff
@@ -0,0 +1,158 @@
+/*eslint no-console: 0*/
+import getName from './get-name';
+import { Collection, Model } from 'ember-cli-mirage';
+
+export default function getAll(schema, request){
+  //turn /api/programyears?limit=1 into 'programYears'
```

Comment:
please remove comment.

Reply:
This is a comment about the purpose of the regex.